shaders/src/light_indirect.fs

```diff
@@ -203,21 +194,122 @@ float computeSpecularAO(float NoV, float ao, float roughness) {
  * - The reflected vector may be modified to point towards the dominant specular
  *   direction to match reference renderings when the roughness increases
  */
-vec3 getReflectedVector(const PixelParams pixel, const vec3 n) {
+
+vec3 getReflectedVector(const vec3 v, const PixelParams pixel, const vec3 n) {
```

Comment:
Can you pass parameters like this: PixelParams pixel, vec3 v, vec3 n? In other functions, PixelParams is always first